docs/source/cli_help/anno.rst

```diff
@@ -0,0 +1,42 @@
+.. role:: bash(code)
+
+.. _anno:
+
+anno
+----
+Usage:
+
+.. code-block:: shell
+
+    git machete anno [-b|--branch=<branch>] [<annotation text>]
+    git machete anno -H|--sync-github-prs
+
+If invoked without any argument, prints out the custom annotation for the given branch (or current branch, if none specified with :bash:`-b/--branch`).
```

Comment:
For clarity, I'd replace `argument` with `<annotation text>`... since `argument` can as well correspond to `-b` option